### PR TITLE
ci(Postman): Add refund scenarios for partially captured payment

### PR DESCRIPTION
**Pull request by Gnanasundari24**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description

1. Added refund scenarios for partially captured payment.
2.Addressed the issue in the collections related to the card network, which was causing automation scenarios to fail due to the changes introduced in this PR: (https://github.com/juspay/hyperswitch/pull/3036)"

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?

Ran the collection using newman

Stripe
Screenshot 2023-12-07 at 5 03 01 PM

Bluesnap
Screenshot 2023-12-07 at 5 08 53 PM

Checkout
Screenshot 2023-12-07 at 5 11 20 PM

Adyen
Screenshot 2023-12-07 at 5 13 29 PM

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: postman/collection-dir/bluesnap/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/.meta.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "childrenOrder": [
+    "Payments - Create",
+    "Payments - Capture",
+    "Payments - Retrieve",
+    "Refunds - Create",
+    "Refunds - Create-copy",
+    "Refunds - Retrieve Copy",
+    "Refunds - Validation should throw",
+    "Payments - Retrieve Copy"
```
Comment (pixincreate) [inline postman/collection-dir/bluesnap/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/.meta.json:10]: Can we have better naming for these?

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/.meta.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "childrenOrder": [
+    "Payments - Create",
+    "Payments - Capture",
+    "Payments - Retrieve",
+    "Refunds - Create",
+    "Refunds - Create-copy",
+    "Refunds - Retrieve Copy",
+    "Refunds - Validation should throw",
+    "Payments - Retrieve Copy"
```
Comment (pixincreate) [inline postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/.meta.json:10]: Same here. Can we have naming convention? Something like:
```
Refunds - Create x1
Refunds - Retrieve
Refunds - Validation
Refunds - Retrieve x1
```

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/Refunds - Create-copy/event.test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    function () {
+      pm.expect(jsonData.status).to.eql("succeeded");
+    },
+  );
+}
+
+// Response body should have value "6540" for "amount"
+if (jsonData?.status) {
+  pm.test(
+    "[POST]::/refunds - Content check if value for 'amount' matches '6540'",
```
Comment (pixincreate) [inline postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/Refunds - Create-copy/event.test.js:45]:
```suggestion
// Response body should have value "2000" for "amount"
if (jsonData?.status) {
  pm.test(
    "[POST]::/refunds - Content check if value for 'amount' matches '2000'",
```

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/Refunds - Create/event.test.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+if (jsonData?.status) {
+  pm.test(
+    "[POST]::/refunds - Content check if value for 'status' matches 'succeeded'",
+    function () {
+      pm.expect(jsonData.status).to.eql("succeeded");
+    },
+  );
+}
+
+// Response body should have value "6540" for "amount"
```
Comment (pixincreate) [inline postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario28-Create partially captured payment with refund/Refunds - Create/event.test.js:42]:
```suggestion
// Response body should have value "4000" for "amount"
```

**Review by likhinbopanna — APPROVED** [review]

**Pull request merged**